modules/lfs/server.go

```diff
@@ -91,6 +99,7 @@ func isOidValid(oid string) bool {
 // ObjectOidHandler is the main request routing entry point into LFS server functions
 func ObjectOidHandler(ctx *context.Context) {
 	if !setting.LFS.StartServer {
+		log.Debug("Attempt to access LFS server but LFS server is disabled")
```

Comment:
```suggestion
		log.Info("Attempt to access LFS server but LFS server is disabled")
```

Reply:
I dunno I think Info is too high - there's no error here except on the user's behalf and the router logs will clearly indicate a 404 for the lfs pages.